prowler/providers/m365/services/entra/entra_service.py

```diff
@@ -585,6 +592,7 @@ class User(BaseModel):
     on_premises_sync_enabled: bool
     directory_roles_ids: List[str] = []
     is_mfa_capable: bool = False
+    account_enabled: bool = True
```

Comment:
```suggestion
    account_enabled: bool = True
```
Can we set this to False by default?

Reply:
I think it’s better to set it to true since most accounts are enabled, and in the worst-case scenario I’d rather have a false positive than a false negative (in this case, a ghost finding).